src/Sylius/Behat/Context/Transform/OrderContext.php

```diff
@@ -47,6 +47,18 @@ public function getOrderByNumber($orderNumber)
         return $order;
     }
 
+    /**
+     * @Transform /^latest order$/
+     */
+    public function getLatestOrder()
+    {
+        $orders = $this->orderRepository->findLatest(1);
+
+        Assert::notNull($orders[0],'No order have been made');
```

Comment:
```suggestion
        Assert::notEmpty($orders,'No order have been made');
```

otherwise, an Error will be thrown, as an item with this index does not exist.